robosats/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: ${APP_ROBOSATS_IP}
+      APP_PORT: ${APP_ROBOSATS_PORT}
+      ROBOSATS_ONION: robosats6tkf3eva7x2voqso3a5wcorsnw34jveyxfqi2fu7oyheasid.onion
+
+  web:
+    image: recksato/robosats-client:latest
```

Comment:
For security reasons, `latest` isn't accepted. A specific version should be tagged, pinned with the checksum of the image.

Reply:
Done